azurerm/internal/services/policy/client/client.go

```diff
@@ -2,7 +2,7 @@ package client
 
 import (
 	"github.com/Azure/azure-sdk-for-go/services/preview/policyinsights/mgmt/2019-10-01-preview/policyinsights"
-	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-09-01/policy"
+	"github.com/Azure/azure-sdk-for-go/services/preview/resources/mgmt/2020-03-01-preview/policy"
```

Comment:
why are we moving to a Preview API, it's been 8 months since then, is there not a stable release available?

Reply:
This is quite awkward that part in this package is stable and other part in this package is preview.
Policy assignment, definition and set-definition in this package are using stable api-version 2020-03-01, and policy exemption is using api-version 2020-07-01-preview and this resource currently does not have stable api-versions yet.
The "displayed" api-version in a go SDK package is usually decided by the service team (how they name their readme tag), therefore this package is named as 2020-03-01-preview.
Also I have changed the description, this upgrade is prerequisite to support the policy exemption.